src/memgraph.cpp

```diff
@@ -524,6 +566,12 @@ int main(int argc, char **argv) {
 #endif
   }
 
+  auto *maybe_ha_init_file = std::getenv(kMgHaClusterInitQueries);
```

Comment:
Maybe just write comment that this must be done after creating `CoordinatorState` so someone doesn't make a mistake when refactoring